pkg/external-boot-image/Dockerfile

```diff
@@ -0,0 +1,8 @@
+# syntax=docker/dockerfile-upstream:1.5.0-rc2-labs
+
+FROM lfedge/eve-xen-tools:e36677ad441c2d5cd88cdd9831959eaa4715ab82 as initrd-build
+FROM docker.io/lfedge/eve-kernel:eve-kernel-amd64-v6.1.38-generic-d2da815271a2-gcc as  kernel-build
```

Comment:
You are fixing the kernel architecture + version here, which breaks the compatibility with arm64... so far I can build `make ZARCH=arm64 HV=kubevirt eve-kube/eve/etc` so you need to check the architecture here and pick the right kernel, or at least forbid the build of kubevirt variant for arm64....

Reply:
For now we have no plans to support in arm64. But I agree we need to have a better of of handling that ? I am thinking of actually passing the KERNEL_TAG to the docker file as env variable in that way architecture will always be correct. I will look into that.

Reply:
yes, I agree, use KERNEL_TAG would be better....

Reply:
@rene do you know how to pass env variables like KERNEL_TAG to linuxkit  pkg build ?  docker build seems to have an option but not linuxkit build.

Reply:
Ok I will go ahead and add a script under tools to process the KERNEL_TAG and as well as XENTOOLS_TAG for pkg external-boot-image